plugins/org.jkiss.dbeaver.ext.databend/plugin.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.2"?>
+
+<plugin>
+
+    <extension point="org.jkiss.dbeaver.generic.meta">
+        <meta id="databend"
+              class="org.jkiss.dbeaver.ext.databend.model.DatabendMetaModel"
+              driverClass="com.databend.jdbc.DatabendDriver"
+              dialect="databend"/>
+    </extension>
+
+    <extension point="org.jkiss.dbeaver.dataSourceProvider">
+        <datasource
+                class="org.jkiss.dbeaver.ext.databend.DatabendDataSourceProvider"
+                description="Databend"
+                id="databend"
+                parent="generic"
+                label="Databend"
+                icon="icons/databend_icon.png"
+                dialect="databend">
+            <drivers managable="true">
+                <driver
+                    id="databend"
+                    label="Databend"
+                    class="com.databend.jdbc.DatabendDriver"
+                    sampleURL="jdbc:databend://{username}:{password}@{host}:{port}[/{database}]"
+                    defaultPort="8000"
+                    icon="icons/databend_icon.png"
+                    iconBig="icons/databend_icon_big.png"
+                    description="Databend driver (server version 1.0.0+)"
+                    supportedConfigurationTypes="MANUAL,URL"
+                    webURL="https://github.com/databendcloud/databend-jdbc"
+                    categories="sql,analytic,columns">
+                    <file type="jar" path="maven:/com.databend:databend-jdbc:RELEASE[0.0.7]" load-optional-dependencies="true"/>
```

Comment:
But why do you have [0.0.7]? https://mvnrepository.com/artifact/com.databend/databend-jdbc I can not see this version on the maven repository
Probably removing a specific version will help.

Reply:
Sorry, I wrote a wrong version and I have released 0.0.7 just now https://central.sonatype.com/artifact/com.databend/databend-jdbc/0.0.7

Reply:
Ok, thanks